Rakefile

```diff
@@ -68,7 +84,20 @@ task :chdir_pkg => [:package_dirs] do
 end
 
 task :deb => [:chdir_pkg, :propsd_source] do
-  sh "fpm --deb-no-default-config-files --deb-recommends nodejs -s dir -t deb -n \"#{name}\" -v #{version} opt/"
+  command = [
+    'fpm',
+    '--deb-no-default-config-files',
+    "--depends \"nodejs = #{target_version}\"",
```

Comment:
There is *ish* by doing `--depends "nodejs >= 4" --depends "nodejs << 5"` but I have two concerns:
1. `dpkg` doesn't respect semver afaik so saying `>=4` might just pick up `4.0.0` or `4.0.x` or `4.x.x` depending on what the ppa is serving.
1. Pegging the depends version to our development version at least guarantees that we're deploying with what we've tested against.